projects/workflows-creator/src/lib/workflow-builder.module.ts

```diff
@@ -167,6 +172,8 @@ import {LocalizationPipe} from './pipes/localization.pipe';
     {provide: LINK_OR_GATEWAY_STRATEGY, useClass: OrGatewayLinkStrategy},
     {provide: LINK_NONE_STRATEGY, useClass: NoLinkStrategy},
     {provide: CONDITION_LIST, useValue: typeTuppleList},
+    {provide: ENV_TOKEN, useValue: 'local'},
+    LocalizationPipe,
```

Comment:
Why is this provided at both module and component levels?

Reply:
![Screenshot from 2023-07-10 21-15-18](https://github.com/sourcefuse/workflows-creator/assets/109737499/dfbfdbae-3c93-4b96-9933-a84e6268a210)
It will give this error if i remove from module .

Reply:
So solve the bug na? You should not have to bind this at both levels, it should be provided at one of the two levels only